cluster/aws/util.sh

```diff
@@ -1498,6 +1503,12 @@ function wait-master() {
     fi
     printf "."
     sleep 5
+    attempt=$(($attempt+1))
+    if (( attempt > 240 )); then
+      echo
+      echo "Waiting for master failed after 20 minutes. There might be some uncaught error during start."
```

Comment:
20 minutes are too long, it should be done within 5 minutes.

Reply:
Updated.